emulators/springcoat-vm/src/main/java/cc/squirreljme/vm/springcoat/MLEPencil.java

```diff
@@ -321,6 +440,46 @@ public Object handle(SpringThreadWorker __thread, Object... __args)
 			}
 		}
 	},
+
+	/**
+	 * {@link PencilShelf#hardwareFillRoundRect(PencilBracket, int, int, int,
+	 * int, int, int)}.
+	 */
+	HARDWARE_FILL_ROUND_RECT("hardwareFillRoundRect:" +
+		"(Lcc/squirreljme/jvm/mle/brackets/PencilBracket;IIIIII)V")
```

Comment:
There is the newer and more precise `MLEDispatcher.methodKey()` for declaring method signatures in SpringCoat here.